tools/kind/webhook-proxy.yaml

```
@@ -0,0 +1,68 @@
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: webhook-selfsigned-issuer
+  namespace: kubeflow
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: webhook-serving-cert
+  namespace: kubeflow
+  spec:
+   dnsNames:
+     - ml-pipeline-reverse-proxy.kubeflow.svc
+     - ml-pipeline-reverse-proxy.kubeflow.svc.cluster.local
+   issuerRef:
+      kind: Issuer
+      name: webhook-selfsigned-issuer
+   secretName: webhook-server-cert
```

There is something wrong here. The indentation doesn't seem correct. Should it be the following?

```suggestion
spec:
  dnsNames:
    - ml-pipeline-reverse-proxy.kubeflow.svc
    - ml-pipeline-reverse-proxy.kubeflow.svc.cluster.local
  issuerRef:
    kind: Issuer
    name: webhook-selfsigned-issuer
  secretName: webhook-server-cert
```